plugins/vendor/webex/lib/model/WebexAPIDropFolder.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * @package plugins.WebexAPIDropFolder
+ * @subpackage model
+ */
+class WebexAPIDropFolder extends RemoteDropFolder
+{
+	
+	const WEBEXPAI_VENDOR_INTEGRATION_ID = 'webexapi_vendor_integration_id';
```

Comment:
you wrote WEBEXPAI instead of webexapi
also sometime you use webexapi, sometimes webexAPI and somtimes webex_api lets be consistent